modules/publicize/publicize.php

```diff
@@ -418,15 +419,102 @@ function save_meta( $post_id, $post ) {
 		// Next up will be ::publicize_post()
 	}
 
+	public function update_published_message( $messages ) {
+		global $post_type, $post_type_object, $post;
+		if ( ! $this->post_type_is_publicizeable( $post_type ) ) {
+			return $messages;
+		}
+		$view_post_link_html = '';
+		$viewable = is_post_type_viewable( $post_type_object );
+		if ( $viewable ) {
+			$view_text = __( 'View post' ); // intentionally omitted domain
+
+			if ( 'jetpack-portfolio' == $post_type ) {
+				$view_text = esc_html__( 'View project', 'jetpack' );
+			}
+
+			$view_post_link_html = sprintf( ' <a href="%1$s">%2$s</a>',
+				esc_url( get_permalink( $post ) ),
+				$view_text
+			);
+		}
+
+		$services = $this->get_publicizing_services( $post->ID );
+		if ( empty( $services ) ) {
+			return $messages;
+		}
+
+		$labels = array();
+		foreach ( $services as $service => $display_names ) {
+			$labels[] = sprintf(
+				/* translators: Service name is %1$s, and account name is %2$s. */
+				_x( '%1$s (%2$s)', 'Service and account string', 'jetpack' ),
```

Comment:
Please convert all `_x()` that you added in this PR to just `__()`. The context prevents re-usage of translations and the comment is already helpful.